lib/workflows/fabric-common.js

```diff
@@ -231,6 +231,16 @@ function provisionFabricNats(job, cb) {
                     ticket: tick.ticket.uuid
                 }
             };
+            
+            /* If the request says to place the VM on an encrypted CN then also place the NAT
+             * zone on an encrypted CN. This is necessary when all CNs are encrypted. 
+             * If the user doesn't want the NAT zone on an encrypted CN then they can first create 
```

Comment:
There should be a better way to do this than "1st Fabric-user wins", but I don't know how off the top of my head.  Pardon any delays on this while I think about this.

Reply:
How about new SAPI configuration value for the `nat` service that allows you to specify whether NAT zones are placed on encrypted CNs or not?

Reply:
Actually, I like what @arekinath did on his setup where he modified cnapi/dapi to always be ok with provisioning "unencrypted" things on an encrypted CN. Perhaps a new `cnapi` setting for that? I think that would be better than this because this change is specific to NAT zones but still requires specifying `encryption` for all other VMs when you're dealing with all encrypted CNs.

Reply:
@arekinath's approach is the right way to handle this. The encryption check can be skipped if the zone is a nat zone. It doesn't affect the nat zone itself one way or the other. And tying it to whatever happens to be created first is the wrong thing.

I'm going to close this in favor of something done via sdc-designation.

Reply:
@bahamat Sounds good. I'll take a look at `sdc-designation` and see what would need to be done there if no one else will be looking at this.